spec/expected/version.txt

```diff
@@ -0,0 +1 @@
+4.7.3
```

Comment:
maybe we can read out the dynamic value from package.json to make updating less annoying

Reply:
That would really be preferable! It's possible to do require('path/to/package.json').version

Reply:
Might require reworking the task runner but definitely doable and a better approach.

Reply:
another approach might be just verifying that the output is `/\d+\.\d+\.\d+/` but I wouldn't worry about missing a test for this one command if it's too fiddly. 

Main thing is that having this fail after every release would be a major maintenance burden 😅

Reply:
Like most things, this ended up being pretty straight forward 🥳.

Reply:
This file can be removed now.